### PR TITLE
Add a description to calcExperience

### PR DESCRIPTION
**Pull request by LeoDog896**
### Description
This improves the quality of the documentation at least a little, allowing the user reading the documentation to understand the calcExperience function a little more.
---

### CODE REVIEW

File: src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java [hunk 1]
```diff
@@ -404,6 +404,7 @@ else if (level >= 16 && level <= 30) // Truncating decimal parts probably works
 			}
 			
 		}.description("Calculates the total amount of experience needed to achieve given level from scratch in Minecraft.")
+				.examples("calcExperience(3) # Will return \"27\"", "send \"The max capacity of your level is %calcExperience(%level of player%)%\" to the player")
```
Comment (Matocolotoe) [inline src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java]:
```suggestion
				.examples("calcExperience(3) # Will return \"27\"", "send \"Your current level is %calcExperience(level of player)%\" to the player")
```

Comment (Blueyescat) [reply src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java]: should calculate including player's current exp

Comment (Matocolotoe) [reply src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java]: oops yes

**Comment by ShaneBeee** [comment]
Just wondering where we're at with this?
I see that there have been some requested changes, but this PR seems to have gone stale about 5 months ago.

@LeoDog896 were you planning on updating this or shall we close it?

**Comment by ShaneBeee** [comment]
> Just wondering where we're at with this?
> I see that there have been some requested changes, but this PR seems to have gone stale about 5 months ago.
> 
> @LeoDog896 were you planning on updating this or shall we close it?

In response to my last message (15 days ago), I see no progress has been made on this PR.
Closing this due to changes requested and inactivity by the PR author.

If you wish to re-open in the future, please do, as we always appreciate contributions.

**Pull request closed without merging**